velox/common/memory/MemoryPool.h

```diff
@@ -43,14 +43,28 @@ namespace facebook::velox::memory {
       "{}",                                                         \
       errorMessage);
 
+/// This macro shall ideally only be used externally outside of MemoryPool,
+/// because poolFailMessage requires an internal lock. If used internally
+/// please use the following _VELOX_MEM_POOL_ABORTED macro that does not produce
+/// detailed pool info. If one has to use this one internally caution of double
+/// locking should be excercised
 #define VELOX_MEM_POOL_ABORTED(pool)                                \
```

Comment:
Can you verify if kMemAborted cause insufficient error shown at coordinator side or not? If not, maybe we shall just use VELOX_MEM_POOL_CAP_EXCEEDED
And we can just pass the error message to this macro to avoid the locking issue.

Reply:
No kMemAborted is not added to the exception translation layer, along with a couple of other errors. We shall have a follow up of cleaning it up.